mysql-test/suite/rpl/r/percona_log_slow_replica_statements_row.result

```diff
@@ -0,0 +1,66 @@
+include/master-slave.inc
+Warnings:
+Note	####	Sending passwords in plain text without SSL/TLS is extremely insecure.
+Note	####	Storing MySQL user name or password information in the master info repository is not secure and is therefore not recommended. Please consider using the USER and PASSWORD connection options for START SLAVE; see the 'START SLAVE Syntax' in the MySQL Manual for more information.
+[connection master]
+SET SESSION binlog_rows_query_log_events=1;
```

Comment:
I think, this looks a requirement for query logging SQL text to appear. Can you clarify in commit message and github description.

Also, how will the content looks like if this is turned off? possible to add a test sceanrio